changelog_unreleased/handlebars/pr-9403.md

```diff
@@ -0,0 +1,29 @@
+#### Fix unstable newlines after `<Textarea />` (#9403 by @fisker, fixes in `simple-html-tokenizer` by @dcyriller)
```

Comment:
credit goes to Robert ^^ (in https://github.com/tildeio/simple-html-tokenizer/pull/90)
```suggestion
#### Fix unstable newlines after `<Textarea />` (#9403 by @fisker, fixes in `simple-html-tokenizer` by @rwjblue)
```